modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -47,64 +54,56 @@ You can use the information in this topic to publish Mule 3 and Mule 4 assets in
 If you are publishing your asset to the master organization, the group ID is the same as the organization ID.
 The URL format is:
 +
-`+https://maven.anypoint.mulesoft.com/api/v1/organizations/ORGANIZATION_ID/maven+`
+`+https://maven.anypoint.mulesoft.com/api/v3/organizations/ORGANIZATION_ID/maven+`
 +
 In the EU, use:
```

Comment:
Do we mention earlier that there is a different URL format in the EU, as opposed to...the rest of the world? the US?